cfme/common/vm.py

```diff
@@ -391,6 +391,8 @@ def load_details(self, refresh=False):
         else:
             if refresh:
                 toolbar.refresh()
+                # Workaround while BZ 1390157 is still open
+                sel.click(self.find_quadicon())
```

Comment:
updated this to use BZ blocker and put main BZ number( previous was marked as Duplicate)